### PR TITLE
bump images for event-bus and kubeless

### PR DESCRIPTION
**Pull request by k15r**
**Description**

Changes proposed in this pull request:

- bump all event-bus and kubeless related images

**Related issue(s)**

### CODE REVIEW

File: resources/application-connector/values.yaml [hunk 1]
```diff
@@ -38,11 +38,11 @@ global:
     dir:
     version: fbf1000b
   event_service:
-    dir: develop/
-    version: e029fcf4
+    dir:
```
Comment (montaro) [inline resources/application-connector/values.yaml]: do you think we still need this key? should we remove it?

File: resources/nats-streaming/values.yaml [hunk 2]
```diff
@@ -16,8 +16,8 @@ global:
   containerRegistry:
     path: eu.gcr.io/kyma-project
   event_bus:
-    dir: develop/
-    version: 5e4264e9
+    dir: ""
```
Comment (montaro) [inline resources/nats-streaming/values.yaml]: In the other occurrences, you did not use the `""`, is it safe to use both styles?

File: resources/application-connector/values.yaml (hunk 1 above)
Comment (k15r) [reply resources/application-connector/values.yaml]: it is still referenced in the templates, so we cannot just remove it from the values

File: resources/nats-streaming/values.yaml (hunk 2 above)
Comment (k15r) [reply resources/nats-streaming/values.yaml]: correct me if im wrong: "" == empty string , without the "" it evaluates to null... the result is the same

File: resources/application-connector/values.yaml (hunk 1 above)
Comment (montaro) [reply resources/application-connector/values.yaml]: Yes, I understand, would it be cleaner if we removed it from the templates as well?

File: resources/nats-streaming/values.yaml (hunk 2 above)
Comment (montaro) [reply resources/nats-streaming/values.yaml]: Yes, sure, but for the sake of consistency, should we make all of them with the same value?
My concern was only if the template engine would panic if he found the empty string, but fine forget about it 👍

Comment (antoineco) [reply resources/nats-streaming/values.yaml]: The templating language does the right thing in both case: generate an emtpy string.

Comment (antoineco) [reply resources/nats-streaming/values.yaml]: The templating language does the right thing in both cases: generate an emtpy string.

**Pull request merged**